src/pykiso/test_setup/dynamic_loader.py

```diff
@@ -284,9 +284,9 @@ def get_instance(self, name: str) -> AuxiliaryCommon:
         # of the current auxiliary
         start_method = getattr(inst, "start", None)
 
-        # TODO: remove it after all auxes are adapted to DTAuxiliaryInterface
+        # TODO: remove it after all auxes are adapted to AuxiliaryInterface
```

Comment:
We can remove this whole block now also :) 
(the comment + the id_dt check + the elif)